hypervisor/arch/x86/guest/virtual_cr.c

```diff
@@ -140,7 +172,10 @@ static void vmx_write_cr0(struct acrn_vcpu *vcpu, uint64_t cr0)
 				vcpu_set_efer(vcpu, vcpu_get_efer(vcpu) | MSR_IA32_EFER_LMA_BIT);
 			} else if (is_pae(vcpu)) {
 				/* enabled PAE from paging disabled */
-				load_pdptrs(vcpu);
+				if (load_pdptrs(vcpu) != 0) {
+					err_code = -EFAULT;
```

Comment:
-EFAULT is not used later on.
err_code ? or bool err_found? or load_failure? or failure? bool is better

Reply:
ok. thanks